Community/Tdarr_Plugin_MC93_Migz1FFMPEG.js

```diff
@@ -9,9 +9,11 @@ const details = () => ({
                   Settings are dependant on file bitrate
                   Working by the logic that H265 can support the same ammount of data at half the bitrate of H264.
                   NVDEC & NVENC compatable GPU required.
-                  This plugin will skip any files that are in the VP9 codec.`,
-  Version: '3.1',
-  Tags: 'pre-processing,ffmpeg,video only,nvenc h265,configurable',
+                  This plugin will skip any files that are in the VP9 codec.
+                  This Plugin makes use of "nvidia-smi" if it is accessible from the node.
+                  Measures GPU utilization from nvidia-smi and assigns task to GPU with least GPU utilization.`,
+  Version: '3.2',
+  Tags: 'pre-processing,ffmpeg,video only,nvenc h265,configurable,NVIDIA,GPU,multiple GPUs',
```

Comment:
Tags are invalid. Keep them to spec as listed here: https://github.com/HaveAGitGat/Tdarr_Plugins/blob/192a8aa0273e4572f65683d71787a12e284c150e/examples/Tdarr_Plugin_pre1_Pre_Proc_Example.js#L22

Reply:
Didn't know there was a list of Tags. Will update them!